src/style.css

```diff
@@ -0,0 +1,116 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.page {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+}
+
+/* #region card */
+.card {
+  position: absolute;
+  justify-content: center;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: #fff;
+}
+
+.card__photo-imac {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 36px;
+}
+
+.card__about-product {
+  margin: 0 16px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__about-code {
+  margin: 4px 16px 0;
+  color: #616070;
+}
+
+.card__reviews {
+  display: flex;
+  flex-direction: row-reverse;
+  margin: 18px 16px 0;
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin: 24px 16px 0;
+}
+
+.card__price-title {
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__price-count {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  margin: 16px 0;
+}
+
+.card__button-link {
+  padding: 11px 69px;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #fff;
+  background-color: #00acdc;
+  border-radius: 5px;
+  border: 1px solid #00acdc;
+}
+
+.card__button-link:hover {
+  background-color: #fff;
+  color: #00acdc;
+  border: 1px solid #00acdc;
+}
+
+/* #endregion */
+
+/* #region stars-block */
+.stars {
+  display: flex;
+  position: relative;
+  margin-left: 17px;

```

Comment:
position, margins, size are external styles
you can't use them for a bem block
use mix (check the example in the checklist)
<img width="1008" alt="image" src="https://user-images.githubusercontent.com/102411130/203507643-9bb8dedb-006e-4a4a-a7d6-e15bdc3e3d6d.png">